tests/test_runner/test_discover_runner.py

```diff
@@ -188,3 +190,12 @@ def test_exclude_tags(self):
         self.assertEqual(runner.build_suite(['test_discovery_sample.tests_sample']).countTestCases(), 0)
         runner = DiscoverRunner(exclude_tags=['slow'])
         self.assertEqual(runner.build_suite(['test_discovery_sample.tests_sample']).countTestCases(), 4)
+
+    @unittest.skipUnless(six.PY34, "subtests were added in Python 3.4")
+    def test_subtest(self):
+        # This test is primarily for checking that subtests work when running
+        # tests with the --parallel option:
+        # https://code.djangoproject.com/ticket/26942
```

Comment:
We only use ticket references for obscure issues that benefit from the additional context of the ticket and even then only in the form "#26942", not a complete URL.